src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsEnforcementSpec.groovy

```diff
@@ -473,4 +473,32 @@ class PriceFloorsEnforcementSpec extends PriceFloorsBaseSpec {
         assert response.seatbid?.first()?.bid?.collect { it.id } == [bidResponse.seatbid.first().bid.first().id]
         assert response.seatbid.first().bid.collect { it.price } == [floorValue]
     }
+
+     def "PBS floors shouldn't enforce when ext.prebid.floors = false or config.auction.priceFloors.enabled = false"() {
```

Comment:
1) There should not be `or` in the test. You have actual and discreet condition.
2) Your test is parametrized, please use placeholders instead of static `false` and `true`.